teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>typo: to <strong>simulate, in</strong> this case</p><p> </p><p>Doesn't pdbfixer already add hydrogens?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>As of now we skip the hydrogens in the fixing step and add them here. Is it possible to evaluate which option is better?</p>

Reply:
Just checked, pdbfixer is using OpenMM's addhydrogens functionality, so there is no difference :), but we only need to do it once.